lib/components/user/notification-prefs-pane.js

```diff
@@ -123,7 +114,7 @@ const NotificationPrefsPane = ({
                 onChange={handleChange}
                 value={type}
               >
-                {text}
+                <FormattedMessage id={`components.NotificationPrefsPane.${type}Select`} />
```

Comment:
Please do not create dynamic message IDs as it is hard to automatically keep track of which IDs are in the code. Instead, please create a hardcoded message ID string for each possible message.